apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractR4ResourceProvider.java

```diff
@@ -277,25 +278,39 @@ public T read(@IdParam IdType claimId, RequestDetails requestDetails) {
   @Trace
   public Bundle findByPatient(
       @RequiredParam(name = "mbi")
-          @Description(shortDefinition = "The patient identifier to search for")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PAC_MBI,
+              value = OpenAPIContentProvider.PAC_MBI_VALUE)
           ReferenceParam mbi,
       @OptionalParam(name = "type")
-          @Description(shortDefinition = "A list of claim types to include")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_CLAIM_TYPE_SHORT,
+              value = OpenAPIContentProvider.EOB_CLAIM_TYPE_VALUE)
           TokenAndListParam types,
       @OptionalParam(name = "startIndex")
-          @Description(shortDefinition = "The offset used for result pagination")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_START_INDEX_SHORT,
+              value = OpenAPIContentProvider.PATIENT_START_INDEX_VALUE)
           String startIndex,
       @OptionalParam(name = "isHashed")
-          @Description(shortDefinition = "A boolean indicating whether or not the MBI is hashed")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PAC_IS_HASHED,
+              value = OpenAPIContentProvider.PAC_IS_HASHED_VALUE)
           String hashed,
       @OptionalParam(name = "excludeSAMHSA")
-          @Description(shortDefinition = "If true, exclude all SAMHSA-related resources")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_SHORT,
+              value = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_VALUE)
           String samhsa,
       @OptionalParam(name = Constants.PARAM_LASTUPDATED)
-          @Description(shortDefinition = "Include resources last updated in the given range")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_LAST_UPDATED_SHORT,
+              value = OpenAPIContentProvider.PATIENT_LAST_UPDATED_VALUE)
           DateRangeParam lastUpdated,
       @OptionalParam(name = "service-date")
-          @Description(shortDefinition = "Include resources that completed in the given range")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_SERVICE_DATE_SHORT,
+              value = OpenAPIContentProvider.EOB_SERVICE_DATE_VALUE)
```

Comment:
I'm not sure the EOB message can be used unchanged here since it is referencing `thru` which isn't a field in the PACA response.